src/components/Card.jsx

```diff
@@ -0,0 +1,56 @@
+import { dateDiffCalc, dateFormatter } from "../util/datecalc";
+import emptyLogo from "../images/emptylogo.svg";
+import kebab from "../images/kebab.svg";
+import useToggle from "../hooks/useToggle";
+import star from "../images/star.svg";
+
+function SelectMenu() {
+  return (
+    <div className="selectMenu-container">
+      <div className="selectMenu">삭제하기</div>
+      <div className="selectMenu clicked">폴더에 추가</div>
+    </div>
+  );
+}
+
+function Card({ link }) {
+  const [kebabValue, KebabTogglefunc] = useToggle(false);
+  const cardImage =
+    link.imageSource || link["image_source"]
+      ? link.imageSource || link["image_source"]
+      : emptyLogo;
```

Comment:
link.imageSource || link["image_source"] 로 나타내는 이유가 있을까요?
```suggestion
  const cardImage =
    link.imageSource ?? emptyLogo;
```

Reply:
받아오는 데이터 키값이 달라서 저런식으로 처리하였습니다.